### PR TITLE
fixes hardcoded network name

### LINKED ISSUE #26: Trade on Testnet: prefix error
Making a trade USDt -> L-BTC trade I got an error about address prefix

image

### PR DESCRIPTION
**Pull request by bordalix**
Fixes #26

@tiero please review

I now have this error on testnet (I have 0.01 L-BTC on my wallet, and I'm trying to trade 0.001 L-BTC for USDT):

Screenshot 2022-01-12 at 09 35 25

Any idea?

### CODE REVIEW

**Comment by louisinger** [comment]
It looks like a compatibility problem with new interface of LDK. @bordalix did u try to bump tdex-sdk ?

**Comment by bordalix** [comment]
Upgraded tdex-sdk to 1.7.9.
Changed `UtxoInterface` to `TxInterface`.

Now I have a new error:

Screenshot 2022-01-12 at 12 18 50

This error happens on every network.

**Comment by louisinger** [comment]
> Upgraded tdex-sdk to 1.7.9. Changed `UtxoInterface` to `TxInterface`. 

`UtxoInterface` is replaced by `UnblindedOutput`: [this PR](https://github.com/vulpemventures/ldk/pull/92) explains the changes about types. However, it seems that the problem comes from marina-provider ([Utxo](https://github.com/vulpemventures/marina-provider/blob/master/src/types.ts#L27) type). Marina provider returns `Utxo` whereas tdex-sdk expects either `UtxoInterface` or `UnblindOutput` (it depends on the version). 

I do think we should use LDK types there in the provider to ensure that tdex-sdk and marina are on the same "types version". cc @tiero ? is there any problem to import ldk in marina provider ?

However, @bordalix  I don't get why u have this error now with that changes, sounds not related :thinking:

File: src/containers/Trade.svelte [hunk 1]
```diff
@@ -32,7 +32,7 @@
   import { showToast } from '../utils/toast';
   import LoadingModal from '../components/LoadingModal.svelte';
   import { getExplorerForNetwork } from '../utils/explorer';
-  import { detectProvider, MarinaProvider } from 'marina-provider';
+  import type { NetworkString } from 'ldk';
```
Comment (louisinger) [inline src/containers/Trade.svelte:35]: Note that tdex-sdk exports all the LDK types, so u can import all the LDK types from tdex-sdk.

Comment (bordalix) [reply src/containers/Trade.svelte]: Commit c352eda fixes this.

**Comment by bordalix** [comment]
> However, @bordalix I don't get why u have this error now with that changes, sounds not related 🤔

Agree, it seems an issue not related with this PR.

**Comment by tiero** [comment]
So let's merge this, will handle this problem in a dedicated issue

**Pull request merged**